vispy/testing/__init__.py

```diff
@@ -40,7 +40,9 @@
 to determine what automatic tests are run.
 """
 
-from ._testing import (SkipTest, requires_application, requires_img_lib,  # noqa
+from ._testing import (SkipTest, requires_application,  # noqa
+                      requires_ipython,   # noqa
```

Comment:
and actually this can fit on the same line as `requires_img_lib` so let's save a line and pop that one up